org.freedesktop.LinuxAudio.Plugins.Cardinal.yml

```diff
@@ -0,0 +1,52 @@
+id: org.freedesktop.LinuxAudio.Plugins.Cardinal
+runtime: org.freedesktop.LinuxAudio.BaseExtension
+sdk: org.freedesktop.Sdk//21.08
+runtime-version: '21.08'
+build-extension: true
+appstream-compose: false
+branch: '21.08'
+build-options:
+  prefix: /app/extensions/Plugins/Cardinal
+  prepend-pkg-config-path: /app/extensions/Plugins/Cardinal/lib/pkgconfig
+  prepend-ld-library-path: /app/extensions/Plugins/Cardinal/lib
+modules:
+  - name: jansson
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/akheron/jansson/releases/download/v2.14/jansson-2.14.tar.bz2
+        sha256: fba956f27c6ae56ce6dfd52fbf9d20254aad42821f74fa52f83957625294afb9
+  - name: Cardinal
+    buildsystem: simple
+    build-commands:
+      - 'make PREFIX=${FLATPAK_DEST} SYSDEPS=true -j $FLATPAK_BUILDER_N_JOBS'
+      - 'make PREFIX=${FLATPAK_DEST} install'
+    post-install:
+      - 'install -d ${FLATPAK_DEST}/lv2/Cardinal.lv2'
+      - 'install -d ${FLATPAK_DEST}/lv2/CardinalFX.lv2'
+      - 'install -d ${FLATPAK_DEST}/lv2/CardinalSynth.lv2'
+      - 'install -d ${FLATPAK_DEST}/vst/Cardinal.vst'
```

Comment:
1. the directory should be `lxvst`,
2. This one is not used

Reply:
the `lxvst` situation still hasn't been fixed??
yikes, you are letting this go out of hand going on like this. please, please, please change this to the normal and expected `vst`.
it can just be a symlink for the time being for compatibility reasons or something, but it is insane that this is still a thing... :scream:

Reply:
This `lxvst` thing should really be fixed, nobody in the world uses this path.

Reply:
no it won't be fixed.

Reply:
jeez. ok then, I feel like I should do something about it.
please stop packaging cardinal

Reply:
@hfiguiere If you are ignoring upstream developers, why should they help with packaging for Flathub?

Reply:
You should talk about this on a thread I'm not involved in for example:
https://github.com/flathub/org.freedesktop.LinuxAudio.BaseExtension/issues/10